kafka-connect-s3/src/main/java/io/confluent/connect/s3/storage/S3OutputStream.java

```diff
@@ -166,6 +163,10 @@ private ObjectMetadata newObjectMetadata() {
 
   private MultipartUpload newMultipartUpload() throws IOException {
     InitiateMultipartUploadRequest initRequest = new InitiateMultipartUploadRequest(bucket, key, newObjectMetadata());
+
+    if (ssea.equals("aws:kms") && StringUtils.isNotBlank(awsKmsKeyId))
+      initRequest.withSSEAwsKeyManagementParams(new SSEAwsKeyManagementParams(awsKmsKeyId));
```

Comment:
This method returns an updated InitiateMultipartUploadRequest rather than mutating the value, so this line should be assigning the result to `initRequest`.

http://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/s3/model/InitiateMultipartUploadRequest.html#withSSEAwsKeyManagementParams-com.amazonaws.services.s3.model.SSEAwsKeyManagementParams-

Reply:
Mutation in place would probably be cleaner here, so replacing this call with `setSSEAwsKeyManagementParams`.